wasmcloud-wasi-kv/wasmcloud-wasi-kv-wit/tests/keyvalue.rs

```diff
@@ -0,0 +1,52 @@
+
+/// Build a fused component for 
+async fn build_fused_component {
+    
+}
+
+#[tokio::test]
+async fn actor_http_log_rng_component() -> anyhow::Result<()> {
+    const WASM: &str = env!("CARGO_CDYLIB_FILE_KEYVALUE");
+    let wasm = wat::parse_file(WASM).context("failed to parse binary")?;
+    let wasm = ComponentEncoder::default()
+        .validate(true)
+        .module(&wasm)
+        .context("failed to encode binary")?
+        .adapter(
+            "wasi_snapshot_preview1",
+            include_bytes!("adapters/wasi_snapshot_preview1.reactor.wasm"),
+        )
+        .context("failed to add WASI adapter")?
+        .encode()
+        .context("failed to encode a component")?;
+    let (wasm, key) = sign(wasm).context("failed to sign component")?;
+
+    let rt = new_runtime();
+    let actor = ActorComponent::new(&rt, wasm).expect("failed to read actor component");
+    assert_eq!(actor.claims().subject, key.public_key());
+
+    let mut actor = actor
+        .instantiate()
+        .await
+        .expect("failed to instantiate actor component");
+
+    let response = actor
+        .call("HttpServer.HandleRequest", Some(REQUEST.as_slice()))
```

Comment:
pretty sure this is copy-pasted from https://github.com/wasmCloud/wasmCloud/pull/276 and not relevant here

Reply:
Yup, these tests need to be removed all together, `wasmtime` doesn't compile properly with some error about `setjmp.h` being missing.

Reply:
:thinking: :thinking: :thinking:

Reply:
I looked and found [this thread](https://github.com/WebAssembly/wasi-sdk/issues/162). I know your branch passes on my machine though, so it doesn't make sense to me but given that I can use your branch to test, I just did that instead.

Reply:
test is gone now, so gonna resolve this!